checker/check-new-request-non-path-default-parameter.go

```diff
@@ -0,0 +1,40 @@
+package checker
+
+import (
+	"github.com/tufin/oasdiff/diff"
+)
+
+func NewRequestNonPathDefaultParameterCheck(diffReport *diff.Diff, _ *diff.OperationsSourcesMap, config Config) Changes {
+	result := make(Changes, 0)
+	if diffReport.PathsDiff == nil || len(diffReport.PathsDiff.Modified) == 0 {
+		return result
+	}
+	for path, pathItem := range diffReport.PathsDiff.Modified {
+		if pathItem.ParametersDiff == nil {
+			continue
+		}
+		for paramLoc, paramNameList := range pathItem.ParametersDiff.Added {
+			if paramLoc == "path" {
+				continue
+			}
+			for _, param := range pathItem.Revision.Parameters {
```

Comment:
Please check that Revision isn't nil before using it

Reply:
Done